blog/2024/keycloak-devday-2025-hackathon-announcement.adoc

```diff
@@ -0,0 +1,41 @@
+:title: Keycloak DevDay 2025 Pre-Conf Event Announcement
+:date: 2024-10-21
+:publish: true
+:author: Sebastian Rose
+
+Keycloak Dev Day 2025 is just around the corner, and we would like to invite you to a special pre-event: the Keycloak Hackathon!
```

Comment:
We should name the event uniquely "DevDay" - without a space between "Dev" and "Day". Please review the whole document for this change.